### DISCUSSION TITLE
08.03(화) Daily Scrum

### DISCUSSION THREAD

**Discussion by ha3158987** [wanted-preonboarding-TEAM2/JARANDA#5] [Scrum]
오늘 진행하실 Task 내용(진행상황, 계획, 문제, 공유하고 싶은 점 등)에 대해 기재해주세요 😃

**Comment by simian114** [comment]
- 제안
  - 글로벌 스타일을 먼저 `develop` 에 올려놓는게 좋을거같습니다!
  - 스크럼 회의를 디스코드에서 말로써 진행하는건 어떨까요? 스크럼기록이라는게 말 그대로 `스크럼 회의` 의 내용, 결과를 적는건데 지금처럼 하면 그냥... 출석부? 가 될거같은 생각이 들어서요.
    -  모든 인원이 모일 필요는 없다고 생각합니다. (모든 인원이면 좋겠지만...) 각 팀당 한명씩만 나와도 진행에 문제는 없을거같아요 :)

**Reply by ha3158987** [reply]
시간만 맞출 수 있다면 제안해주신 방법도 좋을 것 같네요 :)
